pkg/claims/result.go

```diff
@@ -66,3 +69,21 @@ func (r Result) NewOutput(name string, data []byte) Output {
 		Value:         data,
 	}
 }
+
+func (r Result) FilterSensitiveOutputs(result driver.OperationResult, bun bundle.Bundle, store secrets.Store) []Output {
+	o := make([]Output, 0, len(result.Outputs))
+	for name, value := range result.Outputs {
+		sensitive, err := bun.IsOutputSensitive(name)
+		ot := r.NewOutput(name, []byte(value))
+		if err != nil || !sensitive {
```

Comment:
How come this is appending the output if there's an error?

Reply:
Because I'm returning the outputs that doesn't contain any sensitive data back to the caller. I will add documentation about that behavior